app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -80,32 +86,37 @@ fun HomeSectionHeader(
 @Composable
 private fun HomeSectionHeaderContent(
     headerText: String,
-    description: String,
+    textColor: Color = FirefoxTheme.colors.textPrimary,
+    description: String = "",
     showAllTextColor: Color = FirefoxTheme.colors.textAccent,
     onShowAllClick: () -> Unit,
+    hasShowAllText: Boolean,
 ) {
     Row(
         modifier = Modifier.fillMaxWidth(),
     ) {
         SectionHeader(
             text = headerText,
+            textColor = textColor,
             modifier = Modifier
                 .weight(1f)
                 .wrapContentHeight(align = Alignment.Top)
         )
 
-        ClickableText(
-            text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-            modifier = Modifier.padding(start = 16.dp)
-                .semantics {
-                    contentDescription = description
-                },
-            style = TextStyle(
-                color = showAllTextColor,
-                fontSize = 14.sp
-            ),
-            onClick = { onShowAllClick() }
-        )
+        if (hasShowAllText) {
```

Comment:
The nullable approach could then allow the section to be like:
```suggestion
       showAllText?.let {
```

Reply:
I usually avoid using nullable lambdas, but I am good either way in this case.